receiver/sapnetweaverreceiver/README.md

```diff
@@ -0,0 +1,54 @@
+- SAP read-only permission
+- For SAP version 7.38+,  (Windows: execute permission, Unix: write permission) is required for each instance of sapstartsrv on the host. If this permission is being assigned to a group, the - monitoring user in the group must have the group set as primary. If authentication or authorization check fails the request will fail with “Invalid Credentials" or "Permission denied" fault string.
+
+More information on how to setup a SAP NetWeaver Stack for each operating system and version can be found [here](https://help.sap.com/docs/SAP_NETWEAVER/9e41ead9f54e44c1ae1a1094b0f80712/576f5c1808de4d1abecbd6e503c9ba42.html?language=en-US).
+
+## Configuration
+| Field               | Type               | Default                                                                                   | Description                                                                                                                                                  |
+|---------------------|--------------------|-------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| metrics             | map             | (default: see `DefaultMetricsSettings` [here](./internal/metadata/generated_metrics.go) | Allows enabling and disabling specific metrics from being collected in this receiver.                                                                        |
+| endpoint            | string             | `http://localhost:50013`                                                                  | The name of the metric created.                                                                                                                              |
+| username            | string             | `(no default)`                                                                            | Specifies the username used to authenticate using basic auth.                                                                                                |
+| password            | string             | `(no default)`                                                                            | Specifies the password used to authenticate using basic auth.                                                                                                |
+| collection_interval | duration | `10s`                                                                           | This receiver collects metrics on an interval. This value must be a string readable by Golang's [time.ParseDuration](https://pkg.go.dev/time#ParseDuration). |
+
+### Example Configuration
+```yaml
+receivers:
+  sapnetweaver:
+    metrics:
+    endpoint: http://localhost:50013
+    username: root
+    password: password
+    collection_interval: 10s
+processors:
+    batch:
+exporters:
+    googlecloud:
+    logging:
```

Comment:
```suggestion
```

The logging exporter is never used in the telemetry pipeline below, we should probably remove it to avoid confusion

Reply:
removed